pages/system/checkboxes.js

```diff
@@ -51,17 +65,39 @@ _handleChange = (e) => {
         <br />
 
         <System.CodeBlock>
-          {`<System.CheckBox name="six" value={this.state.six} onChange={this._handleChange}>
-  <strong>Unchecked</strong>
-  <br />
-  This CheckBox default is unchecked.
-</System.CheckBox>
+{`const CheckboxUnchecked = () => {
+   return (
+      <div>
+        <CheckBox
+          name="six"
+          value={this.state.six}
```

Comment:
As mentioned earlier, this is my fault, would you be willing to change these state values to something more meaningful?